src/mesh/poly2tri_triangulator.C

```diff
@@ -164,6 +164,11 @@ Poly2TriTriangulator::~Poly2TriTriangulator() = default;
 // Primary function responsible for performing the triangulation
 void Poly2TriTriangulator::triangulate()
 {
+  // We only operate on serialized meshes.  And it's not safe to
+  // serialize earlier, because it would then be possible for the user
+  // to re-parallelize the mesh in between there and here.
+  MeshSerializer _serializer(_mesh);
```

Comment:
Leading underscore naming should be reserved for member variables.

Reply:
Yup.  And this originally *was* a member variable, so I probably just cut and pasted.  Will fix.